src/utils/contacts.ts

```diff
@@ -1,10 +1,22 @@
 import { Platform } from 'react-native'
 import { getAll, getMinimal, MinimalContact } from 'react-native-contacts'
+import {
+  check as checkPermission,
+  RESULTS as PERMISSION_RESULTS,
+  PERMISSIONS,
+} from 'react-native-permissions'
 import Logger from 'src/utils/Logger'
-import { checkContactsPermission } from 'src/utils/permissions'
 
 const TAG = 'utils/contacts'
 
+export const CONTACTS_PERMISSION =
+  Platform.OS === 'ios' ? PERMISSIONS.IOS.CONTACTS : PERMISSIONS.ANDROID.READ_CONTACTS
+
+export async function checkContactPermissionStatusGranted() {
```

Comment:
nit: this name seems a bit verbose. Maybe something like `hasGrantedContactsPermission`?